<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>partsTracker</class>
 <widget class="QDialog" name="partsTracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parts Tracker</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_search">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="asert.qrc">
             <normaloff>:/Images/search.png</normaloff>:/Images/search.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border: 1px solid #a9a9a9;
    padding: 5px;
    border-radius: 10px; /* Adjust the value to control the roundness */
    background-color: #ffffff; /* Set the background color to white */
    color: #333333;
}

QLineEdit:focus {
    border-color: #3e7bb6;
}
</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>478</width>
           <height>58</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_refresh">
         <property name="styleSheet">
          <string notr="true">background-color:transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="asert.qrc">
           <normaloff>:/Images/refresh.png</normaloff>:/Images/refresh.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="styleSheet">
        <string notr="true">QTableView {
    border: 1px solid #d9d9d9;
    background-color: #ffffff;
    padding: 0;
    gridline-color: #d9d9d9;
}

QTableView::item {
    padding: 5px;
    border: none;
}

QTableView::item:selected {
    background-color: #3e7bb6;
    color: #ffffff;
}

QHeaderView::section {
    background-color: #f2f2f2;
    color: #333333;
    padding: 5px;
    font-weight: bold;
    border: none;
    border-right: 1px solid #d9d9d9;
    border-bottom: 1px solid #d9d9d9;
    min-width: 80px;
    min-height: 30px;
    font-size: 18px;
}

QHeaderView::section:checked {
    background-color: #3e7bb6;
    color: #ffffff;
}

QHeaderView::section:pressed {
    background-color: #3e7bb6;
    color: #ffffff;
}

QHeaderView::section:first {
    border-top-left-radius: 5px;
}

QHeaderView::section:last {
    border-top-right-radius: 5px;
    border-right: none;
}

QHeaderView::section:middle {
    border-right: none;
}

QTableView::horizontalHeader {
    border-top: 1px solid #d9d9d9;
    border-bottom: 1px solid #d9d9d9;
}

QTableView::verticalHeader {
    border-right: 1px solid #d9d9d9;
}

QHeaderView::section:horizontal {
    background-color: #e2e2e2;
    border-top: 1px solid #9d9d9d;
}

QHeaderView::section:horizontal:pressed {
    background-color: #3e7bb6;
    color: #ffffff;
}
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="asert.qrc"/>
 </resources>
 <connections/>
</ui>
